VRTweaks/GazeCursor.cs

```diff
@@ -1,30 +0,0 @@
-﻿using HarmonyLib;
```

Comment:
These gaze cursor changes were causing issues with the xbox controller working with menus. They're also seem to no longer be needed after Seaworthy update